### PR TITLE
[doc](typo) fix some typo in table-design section

### PR DESCRIPTION
**Pull request by yagagagaga**
# Versions 

- [x] dev
- [x] 3.0
- [x] 2.1
- [x] 2.0

# Languages

- [x] Chinese
- [x] English

### CODE REVIEW

File: docs/table-design/schema-change.md [hunk 1]
```diff
@@ -97,31 +97,71 @@ ALTER TABLE table_name ADD COLUMN column_name column_type [KEY | agg_type] [DEFA
 
 ### Examples
 
-**1. Adding a key column `new_col` (non-aggregate model) after col1 to `example_rollup_index`**
+#### non-aggregate model
```
Comment (KassieZ) [inline docs/table-design/schema-change.md:100]: `**Non-aggregate Model**`

File: docs/table-design/schema-change.md [hunk 2]
```diff
@@ -97,31 +97,71 @@ ALTER TABLE table_name ADD COLUMN column_name column_type [KEY | agg_type] [DEFA
...
+**2. Adding a value column `new_col` with a default value of 0 after col1 to `example_rollup_index`**
 
 ```sql
 ALTER TABLE example_db.my_table   
 ADD COLUMN new_col INT DEFAULT "0" AFTER col1    
 TO example_rollup_index;
 ```
 
-**3. Adding a Key column `new_col` (aggregate model) after col1 to `example_rollup_index`**
+#### aggregate model
```
Comment (KassieZ) [inline docs/table-design/schema-change.md:136]: `**Aggregate Model**`

File: docs/table-design/schema-change.md [hunk 3]
```diff
@@ -261,6 +318,24 @@ ALTER TABLE table_name ORDER BY (column_name1, column_name2, ...)
 
 ### Example
```
Comment (KassieZ) [inline docs/table-design/schema-change.md:319]: `### Example 1`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/table-design/index/ngram-bloomfilter-index.md [hunk 4]
```diff
@@ -151,6 +151,14 @@ curl --location-trusted -u root: -T amazon_reviews_2014.snappy.parquet -H "forma
 curl --location-trusted -u root: -T amazon_reviews_2015.snappy.parquet -H "format:parquet" http://127.0.0.1:8030/api/${DB}/amazon_reviews/_stream_load
 ```
 
+:::info
+上面的文件可能超过 10 GB，您可能需要调整 be.conf 的 streaming_load_max_mb 防止超过 stream load 文件上传大小的限制，可以通过下面方式动态调整
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/table-design/index/ngram-bloomfilter-index.md:155]: Stream Load

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/table-design/index/ngram-bloomfilter-index.md [hunk 5]
```diff
@@ -151,6 +151,14 @@ curl --location-trusted -u root: -T amazon_reviews_2014.snappy.parquet -H "forma
...
+```bash
+curl -X POST http://{be_ip}:{be_http_port}/api/update_config?streaming_load_max_mb=32768
+```
+需要每台 be 都执行上述命令。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/table-design/index/ngram-bloomfilter-index.md:159]: BE

**Review by gavinchou — APPROVED** [review]

**Pull request merged**